infra/aws/index.ts

```diff
@@ -2052,3 +2073,18 @@ const coreAlbPassportXyz = new aws.lb.ListenerCertificate(
   },
   {}
 );
+
+createV2Api({
```

Comment:
We should get some meaningful outputs here , like : 
- v2 service endpooint 
- docker image tag used